run/testing/sigterm_handler.e2e_test.go

```diff
@@ -35,19 +35,20 @@ func TestSigtermHandlerService(t *testing.T) {
 	defer GetLogEntries(service, t)
 	defer service.Clean()
 
-	requestPath := "/"
+	// Explicitly send SIGTERM
+	requestPath := "/?terminate=1"
 	req, err := service.NewRequest("GET", requestPath)
 	if err != nil {
 		t.Fatalf("service.NewRequest: %v", err)
 	}
 
-	client := http.Client{Timeout: 10 * time.Second}
+	client := http.Client{Timeout: 30 * time.Second}
 	resp, err := client.Do(req)
 	if err != nil {
 		t.Fatalf("client.Do: %v", err)
 	}
-	defer resp.Body.Close()
 	fmt.Printf("client.Do: %s %s\n", req.Method, req.URL)
+	resp.Body.Close()
```

Comment:
What's the reasoning for this change?

Reply:
This was a mistake, and i've restored the defer.
Previously, i was issuing 2 requests, and wanted to ensure Body was closed before proceeding.